numba/tests/test_parfors.py

```diff
@@ -1383,6 +1383,33 @@ def test_impl(n):
         self.check(test_impl, n)
         self.assertEqual(countNonParforArrayAccesses(test_impl, (types.intp,)), 0)
 
+    @skip_parfors_unsupported
+    def test_parfor_array_access_lower_slice(self):
```

Comment:
I think this should go in TestParforsSlice instead of TestParfors.

Reply:
Before making these changes, I want to be sure that we've covered the parallel concerns with these tests.  Is that the case?

Reply:
The basic content of the tests looked fine.